src/screens/DetailDomain/elements/DetailDomainScreenHeader.js

```diff
@@ -110,14 +111,14 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontSize: 12,
     lineHeight: 18,
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
     flexShrink: 1
   },
   point: {
     width: 2,
     height: 2,
     borderRadius: 4,
-    backgroundColor: COLORS.blackgrey,
+    backgroundColor: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the color properties seem to be part of a larger effort to update the color scheme of the application. As long as these changes are consistent with the new color scheme, they should be fine. 



<!-- This is an auto-generated comment by OSS CodeRabbit -->